src/Common/src/Common.Expression/Steeltoe.Common.Expression.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Steeltoe Expression Library</Description>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <AssemblyName>Steeltoe.Common.Expression</AssemblyName>
+    <PackageId>Steeltoe.Common.Expression</PackageId>
+    <PackageTags>NET Core;Expression;</PackageTags>
+  </PropertyGroup>
+
+  <Import Project="..\..\..\..\versions.props" />
+  <Import Project="..\..\..\..\sharedproject.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\Common\Steeltoe.Common.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
```

Comment:
Should we add to versions.props and use a variable?

Reply:
Ya .. good point... will do.

Reply:
I'll look at the TODOs .. several of them are from the Java code .. I just copied them over as part of comments.. I'll update the ones I added

Reply:
Ok once you are done making any more changes, let me know here and I will approve and we can get this merged in.